code-of-conduct.md

```diff
@@ -64,7 +64,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+<aosp-devs@groups.io>
```

Comment:
nitpick: The code changes removes the full stop and every sentence should have a full stop at the end.

Apart from that, looks good.

Reply:
Will fix

Reply:
Fixed in v2.